tests/lowering/mod.rs

```diff
@@ -505,7 +505,7 @@ fn raw_pointers() {
             impl Foo for *i32 { }
         }
         error_msg {
-            "parse error: UnrecognizedToken { token: (30, Token(51, \"i32\"), 33), expected: [\"\\\"const\\\"\", \"\\\"mut\\\"\"] }"
+            "parse error: UnrecognizedToken { token: (30, Token(52, \"i32\"), 33), expected: [\"\\\"const\\\"\", \"\\\"mut\\\"\"] }"
```

Comment:
These tests seem pretty flaky. They seem to break when the parser adds a token. Can the test macro ignore parts of the error (regex?) so that we can keep the `expected` error data while ignoring the specific token IDs?

Reply:
Hmm, I wonder how useful it is to have an error msg here anyways.

Reply:
I can remove everything after `UnrecognizedToken`, or leave that for a later PR?

Reply:
I think we can leave that for a later PR :)